.github/workflows/dapi-grpc.yml

```diff
@@ -15,6 +15,6 @@ on:
 jobs:
   dapi-tests:
     name: Run DAPI gRPC tests
-    uses: dashevo/platform/.github/workflows/test.yml@master
+    uses: dashevo/platform/.github/workflows/test.yml@feat-yarn2
```

Comment:
I think this needs to change to `master` before merge?

Reply:
Yep!

Reply:
I think you can't rename it _before_ merge, as `master` will have the correct file only after the merge? @shumkov

Reply:
Yeah, but before we need to make sure that tests are running, which is not a case atm.